roles/swift-ring/tasks/main.yml

```diff
@@ -1,6 +1,5 @@
 ---
 - name: drop our ring configuration
-  copy: src={{ swift_ring.ring_definition_file }} owner=root group=root
-        dest=/etc/swift/ring_definition.yml mode=644 backup=yes
-  notify: setup swift rings
-
+  copy: src={{ swift_ring.ring_definition_file }} owner=swiftops group=swiftops
```

Comment:
I didn't see issue with copy module during local testing.. I will make sure to verify again... Yeah - have attempted to use template value in all tasks for destination